sdk/push-fcm/src/main/java/com/klaviyo/pushFcm/KlaviyoRemoteMessage.kt

```diff
@@ -23,6 +23,20 @@ import com.klaviyo.core.Registry
  */
 object KlaviyoRemoteMessage {
 
+    /**
+     * Append requisite data from a remote message to an intent
+     * for displaying a notification
+     *
+     * @param message
+     */
+    fun Intent.appendKlaviyoExtras(message: RemoteMessage) = apply {
+        if (message.isKlaviyoMessage) {
+            message.data.forEach {
+                this.putExtra(it.key, it.value)
```

Comment:
was looking at the docstring for `putExtra`:
```
    /**
     * Add extended data to the intent.  The name must include a package
     * prefix, for example the app com.android.contacts would use names
     * like "com.android.contacts.ShowAll".
     *
     * @param name The name of the extra data, with package prefix.
     * @param value The String data value.
     *
     * @return Returns the same Intent object, for chaining multiple calls
     * into a single statement.
     *
     * @see #putExtras
     * @see #removeExtra
     * @see #getStringExtra(String)
     */
    public @NonNull Intent putExtra(String name, @Nullable String value) {
```
do you think the package name prefix part is very important?

Reply:
Good catch!

Reply:
Updated to use namespaced keys `com.klaviyo.*` and only send our keys from the intent also